pillar/heroku/xpro.sls

```diff
@@ -171,6 +174,7 @@ heroku:
     SHEETS_DATE_TIMEZONE: America/New_York
     SHEETS_MONITORING_FREQUENCY: 3600
     SHEETS_TASK_OFFSET: 120
+    SHOW_UNREDEEMED_COUPON_ON_DASHBOARD: {{ env_data.SHOW_UNREDEEMED_COUPON_ON_DASHBOARD }}
```

Comment:
If we want it true in all environments then you can just set it to that value on this line, rather than putting it in the per-env dictionaries. Those are only used for cases where we need a different value between the different environments.